app/services/SessionService.scala

```diff
@@ -100,6 +100,31 @@ class SessionService @Inject()(
       uiJourneySessionDataRepository.set(uiJourneySessionData)
   }
 
+  def setMultipleMongoData(kv: Map[String, String], journeyType: JourneyType)
+                 (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[Throwable, Boolean]] = {
+    val uiJourneySessionData = UIJourneySessionData(hc.sessionId.get.value, journeyType.toString)
+    kv.map { kv_ =>
```

Comment:
is kv_ specific naming convention for tuples? i think there might be a more descriptive name for this